microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/integration/FailSafeCDIModelPropsHealthTest.java

```diff
@@ -0,0 +1,63 @@
+package org.jboss.eap.qe.microprofile.health.integration;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jboss.as.arquillian.api.ServerSetupTask;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientProvider;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+/**
+ * MP Config property is provided by props model option - values are stored in model object.
```

Comment:
Final reminder for improving Javadoc for this set of classes, if you agree.

Reply:
Do you have any suggestions? Properties are set via cli command and are stored directly in WildFly via model.

Reply:
Well, the Javadoc is correct IMO, I was just pointing out a more general issue with documentation with respect to hierarchy, as per my previous comments regarding the other test classes. 
Just a `reminder`, as I stated.